test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/ConcurrentHashMapTest.scala

```diff
@@ -194,6 +194,34 @@ class ConcurrentHashMapTest extends MapTest {
         assertTrue(s"missing value '$value'", encounteredValues.contains(value))
     }
   }
+
+  @Test def addOnKeySetView(): Unit = {
+    val map = factory.empty[String, Int]
+    val keySet = map.keySet(0)
+    assertNull(map.get("ONE"))
+    assertTrue(keySet.add("ONE"))
+    assertEquals("Adding a new key adds the default value.", 0, map.get("ONE"))
+
+    assertFalse(keySet.add("ONE"))
+    assertEquals("Adding an existing key do not alter the value", 0, map.get("ONE"))
```

Comment:
The message here is misleading: Even *if* the value was altered, it would still be `0`. Consider simply dropping this assert.

Reply:
Removed in https://github.com/scala-js/scala-js/pull/4367/commits/1940ba1fc571b40be360be116f980a7f99d9a62b